composables/drop/useHolderOfCollection.ts

```diff
@@ -74,15 +73,9 @@ export default function useHolderOfCollection() {
           id: drop.holder_of,
           account: accountId.value,
         },
-      }).then((res) => (holderOfCollectionData.value = res.data.value)),
-    {
-      watch: [
-        accountId,
-        () => runtimeMintCount,
```

Comment:
How do we deal with the update from this one `runtimeMintCount`? It's not watched then.

Reply:
updated